htdocs/compta/facture/list.php

```diff
@@ -589,7 +589,7 @@
 	$massactionbutton=$form->selectMassAction('', $massaction ? array() : array('presend'=>$langs->trans("SendByMail")));
     
     $i = 0;
-    print '<form method="POST" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
+    print '<form method="GET" name="searchFormList" action="'.$_SERVER["PHP_SELF"].'">'."\n";
```

Comment:
All form must use the POST metho (GET for form is an old behaviour that should be abandonned due to problem, like limits in size by browser or web server, different antispam filtering, url encoding twice, ...)
Why changing this ?